packages/sdk-core/src/model/data/BacktraceStackTrace.ts

```diff
@@ -0,0 +1,12 @@
+export interface BacktraceStackFrame {
+    funcName: string;
+    line?: number;
+    column?: number;
+    sourceCode?: string;
+    library: string;
```

Comment:
By `library`, do you mean `filename`?

Reply:
This is the definition from the api.backtrace.io and represents the thread object we want to send. Backtrace expects filename under the library unless something has changed.

Reply:
done.